core/web/apiv2/observables.py

```diff
@@ -123,19 +168,19 @@ async def bulk_add(request: NewBulkObservableAddRequest) -> list[Observable]:
                 new_observable.value, tags=new_observable.tags
             )
         else:
-            observable = Observable(
-                value=new_observable.value,
-                type=new_observable.type,
-                created=datetime.datetime.now(datetime.timezone.utc),
-            ).save()
-            if new_observable.tags:
-                observable = observable.tag(new_observable.tags)
+            new_observable = new_observable.model_dump()
+            obs_type = new_observable.pop("type")
+            tags = new_observable.pop("tags", [])
+            cls = TYPE_MAPPING[obs_type]
+            observable = cls(**new_observable).save()
```

Comment:
I'd avoid using model_dump() here so we can keep Pydantic's type-hinting.

```suggestion
            tags = new_observable.tags
            cls = TYPE_MAPPING[new_observable.type]
            observable = cls(value=new_observable.value).save()
```

Given that `NewObservableRequest` doesn't support extended attributes.

Reply:
Fixed in 59b5296aa21667a06784b1780ad1d93adc82182a